src/boot/store.js

```diff
@@ -302,6 +302,9 @@ const migrations: {| [string]: (GlobalState) => GlobalState |} = {
     outbox: state.outbox.filter(o => o.sender_id !== undefined),
   }),
 
+  // Add `doNotMarkMessagesAsRead` in `SettingsState`.
+  '30': dropCache,
```

Comment:
This isn't the right thing here; it actually has no effect on the `settings` state. (You asked about this line in a subthread above, too.)

I guess I just said above that I regretted having a previous discussion on a GitHub PR thread instead of in Zulip because it's harder to find here, so I'll make a chat thread to go into more detail :slightly_smiling_face:

Reply:
https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/data.20migrations/near/1230029